examples/hello-world/hw_pre_start.sh

```diff
@@ -15,18 +15,18 @@ export NVFLARE_POC_WORKSPACE
 # prepare POC for 2 clients
 echo "y" | nvflare poc prepare -n 2
 
-if [ ! -L "${NVFLARE_POC_WORKSPACE}/admin/transfer" ]; then
-     echo "'nvflare poc prepare' did not generate symlink '${NVFLARE_POC_WORKSPACE}/admin/transfer'"
+if [ ! -L "${NVFLARE_POC_WORKSPACE}/example_project/prod_00/admin@nvidia.com/transfer" ]; then
+     echo "'nvflare poc prepare' did not generate symlink '${NVFLARE_POC_WORKSPACE}/example_project/prod_00/admin@nvidia.com/transfer'"
      exit 1
 fi
 
 # start POC FL server and FL clients
-nvflare poc start -ex admin
+nvflare poc start -ex admin@nvidia.com
 
-# Check if the FL system is ready until 2 clients are active
-python <<END
-from nvflare.tool.api_utils import wait_for_system_start
-wait_for_system_start(num_clients = 2,  prod_dir = "${NVFLARE_POC_WORKSPACE}")
+# # Check if the FL system is ready until 2 clients are active
+# python <<END
+# from nvflare.tool.api_utils import wait_for_system_start
+# wait_for_system_start(username = "admin@nvidia.com", num_clients = 2,  prod_dir = "${NVFLARE_POC_WORKSPACE}/example_project/prod_00")
 
-END
+# END
```

Comment:
maybe we just remove these commented out lines?

Reply:
don't quite get the purpose of this wait_for ..

Reply:
This is to ensure the system is up and running.  However, the logic does not work all the time.  So it should be removed.